sources/api/schnauzer/src/helpers.rs

```diff
@@ -463,6 +479,43 @@ pub fn ecr_prefix(
     Ok(())
 }
 
+/// `host` takes an absolute URL and trims it down and returns its host.
+pub fn host(
+    helper: &Helper<'_, '_>,
+    _: &Handlebars,
+    _: &Context,
+    renderctx: &mut RenderContext<'_, '_>,
+    out: &mut dyn Output,
+) -> Result<(), RenderError> {
+    trace!("Starting host and port helper");
+    let template_name = template_name(renderctx);
+    check_param_count(helper, template_name, 1)?;
+
+    let url_val = get_param(helper, 0)?;
+    let url_str = url_val
+        .as_str()
+        .with_context(|| error::InvalidTemplateValue {
+            expected: "string",
+            value: url_val.to_owned(),
+            template: template_name.to_owned(),
+        })?;
+    let url = Url::parse(url_str).with_context(|| error::UrlParse {
+        value: url_val.to_owned(),
```

Comment:
```suggestion
        value: url_str,
```
No need to clone the larger JSON object, I think.